packages/terra-disclosure-manager/docs/README.md

```diff
@@ -0,0 +1,209 @@
+# Terra Disclosure Manager
+
+The DisclosureManager is a stateful component used to manage disclosure presentation.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-disclosure-manager`
+
+## Usage
+
+### Implementation Requirements
+
+The DisclosureManager utilizes the AppDelegate API to manage disclosure requests. The components provided as children to the DisclosureManager, and the components disclosed within it, must support an AppDelegate prop (as `app`). This AppDelegate instance will provide component-specific implementations of the various control mechanisms for the manager.
+
+### Rendering
+
+The DisclosureManager does not implement a traditional render function. A `render` prop should be provided to the DisclosureManager in order to render the various components managed by the DisclosureManager. The `render` function should accept an Object parameter containing the DisclosureManager's state.
+
+`render` Argument API:
+
+|Key|Value|
+|---|---|
+|`children`|An Object containing data relative to the children components provided to the DisclosureManager.|
+|`disclosure`|An Object containing data relative to the compoents in the disclosure stack.|
+|`dismissPresentedComponent`|A function that pops the currently disclosed component off the disclosure stack.|
+|`closeDisclosure`|A function that closes the disclosure and removes all components from the disclosure stack.|
+
+`children` Object API:
+
+|Key|Value|
+|---|---|
+|`components`|An Array of React components to render as face-up content. These components were updated with an AppDelegate prop with DisclosureManager integration.|
```

Comment:
Reading this, seems like `were` might be better worded as `are`

Reply:
From what I understand, this is saying The DisclosureManager takes `children` which must be of type object, then, children is passed as:
```
children ={{
    components: [
         <Content1 />,
         <Content2 />,
    ]
}}
``
Should children just take an array of components?